{"uid":"b7a84900cb519e89","name":"Login with invalid credentials test","fullName":"tests.login_test.TestLogin#test_invalid_login","historyId":"9bbc2729137fa409b9bc8067987e1d0d","time":{"start":1707959342126,"stop":1707959352397,"duration":10271},"description":"invalid login","descriptionHtml":"<p>invalid login</p>\n","status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x55d9632e6d93 <unknown>\n#1 0x55d962fca337 <unknown>\n#2 0x55d963015d60 <unknown>\n#3 0x55d963015e51 <unknown>\n#4 0x55d963059bc4 <unknown>\n#5 0x55d96303846d <unknown>\n#6 0x55d96305712d <unknown>\n#7 0x55d9630381e3 <unknown>\n#8 0x55d963008135 <unknown>\n#9 0x55d96300913e <unknown>\n#10 0x55d9632aae4b <unknown>\n#11 0x55d9632aedfa <unknown>\n#12 0x55d9632976d5 <unknown>\n#13 0x55d9632afa6f <unknown>\n#14 0x55d96327b69f <unknown>\n#15 0x55d9632d4098 <unknown>\n#16 0x55d9632d4262 <unknown>\n#17 0x55d9632e5f34 <unknown>\n#18 0x7f0fd4694ac3 <unknown>","statusTrace":"self = <tests.login_test.TestLogin object at 0x7fc5847a62d0>\nemail = 'elias@gmail.com', password = '12345Tr'\ndata = Data(forgot_password=ForgotPassword(error_message=\"We can't find a user with that e-mail address.\", success_message='A...ct='Non Existing', page_title='My Workspace', project_name='test project', project_type='quiz', template_type='Blank'))\n\n    @allure.description(\"invalid login\")\n    @allure.title(\"Login with invalid credentials test\")\n    @pytest.mark.parametrize(\"email, password\", users)\n    @pytest.mark.run(order=3)\n    def test_invalid_login(self, email: str, password: str, data: Data):\n>       self.about_page.click_login_link()\n\ntests/login_test.py:26: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/about_page.py:20: in click_login_link\n    self.click(self.LOGIN_LINK)\npages/base_page.py:60: in click\n    el: WebElement = self.wait.until(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"e8c70f77425518274f326eeebb62e537\")>\nmethod = <function element_to_be_clickable.<locals>._predicate at 0x7fc5844c28e0>\nmessage = ''\n\n    def until(self, method: Callable[[D], Union[Literal[False], T]], message: str = \"\") -> T:\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       #0 0x55d9632e6d93 <unknown>\nE       #1 0x55d962fca337 <unknown>\nE       #2 0x55d963015d60 <unknown>\nE       #3 0x55d963015e51 <unknown>\nE       #4 0x55d963059bc4 <unknown>\nE       #5 0x55d96303846d <unknown>\nE       #6 0x55d96305712d <unknown>\nE       #7 0x55d9630381e3 <unknown>\nE       #8 0x55d963008135 <unknown>\nE       #9 0x55d96300913e <unknown>\nE       #10 0x55d9632aae4b <unknown>\nE       #11 0x55d9632aedfa <unknown>\nE       #12 0x55d9632976d5 <unknown>\nE       #13 0x55d9632afa6f <unknown>\nE       #14 0x55d96327b69f <unknown>\nE       #15 0x55d9632d4098 <unknown>\nE       #16 0x55d9632d4262 <unknown>\nE       #17 0x55d9632e5f34 <unknown>\nE       #18 0x7f0fd4694ac3 <unknown>\n\n.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py:105: TimeoutException","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"base_url","time":{"start":1707959294010,"stop":1707959294010,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"_verify_url","time":{"start":1707959294010,"stop":1707959294010,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"data","time":{"start":1707959294011,"stop":1707959294011,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"description":"invalid login","status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x55d9632e6d93 <unknown>\n#1 0x55d962fca337 <unknown>\n#2 0x55d963015d60 <unknown>\n#3 0x55d963015e51 <unknown>\n#4 0x55d963059bc4 <unknown>\n#5 0x55d96303846d <unknown>\n#6 0x55d96305712d <unknown>\n#7 0x55d9630381e3 <unknown>\n#8 0x55d963008135 <unknown>\n#9 0x55d96300913e <unknown>\n#10 0x55d9632aae4b <unknown>\n#11 0x55d9632aedfa <unknown>\n#12 0x55d9632976d5 <unknown>\n#13 0x55d9632afa6f <unknown>\n#14 0x55d96327b69f <unknown>\n#15 0x55d9632d4098 <unknown>\n#16 0x55d9632d4262 <unknown>\n#17 0x55d9632e5f34 <unknown>\n#18 0x7f0fd4694ac3 <unknown>","statusTrace":"self = <tests.login_test.TestLogin object at 0x7fc5847a62d0>\nemail = 'elias@gmail.com', password = '12345Tr'\ndata = Data(forgot_password=ForgotPassword(error_message=\"We can't find a user with that e-mail address.\", success_message='A...ct='Non Existing', page_title='My Workspace', project_name='test project', project_type='quiz', template_type='Blank'))\n\n    @allure.description(\"invalid login\")\n    @allure.title(\"Login with invalid credentials test\")\n    @pytest.mark.parametrize(\"email, password\", users)\n    @pytest.mark.run(order=3)\n    def test_invalid_login(self, email: str, password: str, data: Data):\n>       self.about_page.click_login_link()\n\ntests/login_test.py:26: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/about_page.py:20: in click_login_link\n    self.click(self.LOGIN_LINK)\npages/base_page.py:60: in click\n    el: WebElement = self.wait.until(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"e8c70f77425518274f326eeebb62e537\")>\nmethod = <function element_to_be_clickable.<locals>._predicate at 0x7fc5844c28e0>\nmessage = ''\n\n    def until(self, method: Callable[[D], Union[Literal[False], T]], message: str = \"\") -> T:\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       #0 0x55d9632e6d93 <unknown>\nE       #1 0x55d962fca337 <unknown>\nE       #2 0x55d963015d60 <unknown>\nE       #3 0x55d963015e51 <unknown>\nE       #4 0x55d963059bc4 <unknown>\nE       #5 0x55d96303846d <unknown>\nE       #6 0x55d96305712d <unknown>\nE       #7 0x55d9630381e3 <unknown>\nE       #8 0x55d963008135 <unknown>\nE       #9 0x55d96300913e <unknown>\nE       #10 0x55d9632aae4b <unknown>\nE       #11 0x55d9632aedfa <unknown>\nE       #12 0x55d9632976d5 <unknown>\nE       #13 0x55d9632afa6f <unknown>\nE       #14 0x55d96327b69f <unknown>\nE       #15 0x55d9632d4098 <unknown>\nE       #16 0x55d9632d4262 <unknown>\nE       #17 0x55d9632e5f34 <unknown>\nE       #18 0x7f0fd4694ac3 <unknown>\n\n.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py:105: TimeoutException","steps":[{"name":"Click Login link","time":{"start":1707959342126,"stop":1707959352396,"duration":10270},"status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x55d9632e6d93 <unknown>\n#1 0x55d962fca337 <unknown>\n#2 0x55d963015d60 <unknown>\n#3 0x55d963015e51 <unknown>\n#4 0x55d963059bc4 <unknown>\n#5 0x55d96303846d <unknown>\n#6 0x55d96305712d <unknown>\n#7 0x55d9630381e3 <unknown>\n#8 0x55d963008135 <unknown>\n#9 0x55d96300913e <unknown>\n#10 0x55d9632aae4b <unknown>\n#11 0x55d9632aedfa <unknown>\n#12 0x55d9632976d5 <unknown>\n#13 0x55d9632afa6f <unknown>\n#14 0x55d96327b69f <unknown>\n#15 0x55d9632d4098 <unknown>\n#16 0x55d9632d4262 <unknown>\n#17 0x55d9632e5f34 <unknown>\n#18 0x7f0fd4694ac3 <unknown>\n\n","statusTrace":"  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/pages/about_page.py\", line 20, in click_login_link\n    self.click(self.LOGIN_LINK)\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/pages/base_page.py\", line 60, in click\n    el: WebElement = self.wait.until(\n                     ^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 105, in until\n    raise TimeoutException(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[{"uid":"4a8d0a9e3e0ea5ec","name":"public ip address","source":"4a8d0a9e3e0ea5ec.txt","type":"text/plain","size":12},{"uid":"456eda0fb6f81bc8","name":"Cookies","source":"456eda0fb6f81bc8.json","type":"application/json","size":3871},{"uid":"56baaf1a7d680df6","name":"Session Storage","source":"56baaf1a7d680df6.json","type":"application/json","size":358},{"uid":"f2d55a03e125802c","name":"Local Storage","source":"f2d55a03e125802c.json","type":"application/json","size":208},{"uid":"422d3559532ea933","name":"Console Logs","source":"422d3559532ea933.json","type":"application/json","size":1521},{"uid":"90bb3d18694af146","name":"Full Page Screenshot","source":"90bb3d18694af146.png","type":"image/png","size":5704608},{"uid":"9ee608315958cd92","name":"URL","source":"9ee608315958cd92.uri","type":"text/uri-list","size":23},{"uid":"69c4f121d56d0556","name":"Network Logs","source":"69c4f121d56d0556.json","type":"application/json","size":23121}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":1,"attachmentsCount":8,"hasContent":true,"attachmentStep":false},"afterStages":[],"labels":[{"name":"epic","value":"Security"},{"name":"severity","value":"blocker"},{"name":"feature","value":"Login"},{"name":"tag","value":"run(order=3)"},{"name":"tag","value":"security"},{"name":"parentSuite","value":"tests"},{"name":"suite","value":"login_test"},{"name":"subSuite","value":"TestLogin"},{"name":"host","value":"fv-az847-865"},{"name":"thread","value":"2375-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.login_test"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"email","value":"'elias@gmail.com'"},{"name":"password","value":"'12345Tr'"}],"links":[],"hidden":false,"retry":false,"extra":{"severity":"blocker","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"tags":["run(order=3)","security"]},"source":"b7a84900cb519e89.json","parameterValues":["'elias@gmail.com'","'12345Tr'"]}